pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -369,6 +369,14 @@ func info(logger log.Logger) *providers.ProviderInfo {
 	info := &providers.ProviderInfo{
 		Rootless: euid != 0,
 	}
+
+	// check if is a local or remote podman
+	// xref: https://github.com/containers/podman/issues/10289
+	err := exec.Command("podman", "--remote", "info").Run()
+	if err == nil {
+		// podman is remote we can't check cgroups information so we exit now
```

Comment:
Alternative PR: https://github.com/kubernetes-sigs/kind/pull/2256

Reply:
Thanks Akihiro, that is much better